### PR TITLE
Added logo, sponsors, forms

### PR DESCRIPTION
**Pull request by GaoYeGithub**

### CODE REVIEW

File: Counterspell Toronto package.json
Comment (samdev-7) [inline Counterspell Toronto package.json:1]: Please don't edit this file and revert your change.

File: Counterspell Toronto yarn.lock
Comment (samdev-7) [inline Counterspell Toronto yarn.lock:1]: Please don't edit this file and revert your change.

File: package-lock.json
Comment (samdev-7) [inline package-lock.json:1]: Please don't edit this file and revert your change.

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Note: When committing, you can choose to exclude certain files!

**Review by samdev-7 — APPROVED** [review]

File: components/Toronto/Locations.js [hunk 1]
```diff
@@ -1,63 +1,34 @@
-import Subscribe from "../Subscribe";
+import Subscribe from "./Subscribe";
```
Comment (samdev-7) [inline components/Toronto/Locations.js:1]: Seems like this change is making the code error. Once this is fixed it's good to be merged!

Comment (samdev-7) [inline components/Toronto/Locations.js:1]: Seems like this change is making the code error. Once this is fixed it's good to be merged!

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by regular030** [comment]
@samdev-7 It should be fixed now

**Review by samdev-7 — APPROVED** [review]

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/toronto**](https://counterspell-q8dbp14i5.hackclub.dev/toronto) |
|:-:|
|<a href="https://counterspell-q8dbp14i5.hackclub.dev/toronto">Screenshot of /toronto</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/MJOvsB-MkI7uWFfzrr0U-59OOcgRBULEHGCn4FRCPeAPVb_HYAvj7IXgQky44YFUYaDmFwtFRaRrws3zOEFhwQ.png">(view full size)</a> |







Commit 9d7d490daf092812c4b8bff35d41e21d9ebeb7e2 (https://counterspell-q8dbp14i5.hackclub.dev).

**Pull request merged**